contracts/NftMarketplace.sol

```diff
@@ -78,6 +81,22 @@ contract NftMarketplace is ReentrancyGuard {
         _;
     }
 
+    // IsNotOwner Modifier - Nft Owner can't buy his/her NFT
```

Comment:
Have the `/*` start from here (rather than ln. 87) as it covers the entire commented-out "block" (as well as being more cleaner).